apollo-router/tests/common.rs

```diff
@@ -582,7 +582,6 @@ impl IntegrationTest {
 
                 let mut request = builder.json(&query).build().unwrap();
                 telemetry.inject_context(&mut request);
-                request.headers_mut().remove(ACCEPT);
```

Comment:
@BrynCooke do you remember why the integration tests were removing the `Accept` header from the request?